src/frontend/hooks/useLastSavedLocation.ts

```diff
@@ -0,0 +1,10 @@
+import {useQuery} from '@tanstack/react-query';
+import {getLastKnownPositionAsync} from 'expo-location';
+
+export const useLastSavedLocation = () => {
+  return useQuery({
+    queryKey: ['lastLocation'],
+    queryFn: async () =>
+      getLastKnownPositionAsync().catch(err => console.log(err)),
```

Comment:
If error-handling or an error object isn't needed for us, maybe return `null` for return type simplification

> Gets the last known position of the device or `null` if it's not available or doesn't match given requirements such as maximum age or required accuracy.

```suggestion
      getLastKnownPositionAsync().catch(err => {
        console.log(err)
        return null
      }),
```

Reply:
no! not needed, was just using it for debugging purposes. Got rid of it here: https://github.com/digidem/CoMapeo-mobile/pull/133/commits/87d01473ddb5ec3bba6f7b1f0e8e278d40d741f3